src/app/create-event/EventInput.tsx

```diff
@@ -0,0 +1,63 @@
+import cx from 'classnames';
+import { FieldErrors } from 'react-hook-form';
+
+import { AriaLiveErrorMessage } from '../../components/AriaLiveErrorMessage';
+
+type InputProps = {
+  register: any;
+  type?: string;
+  label: string;
+  name:
+    | 'name'
+    | 'date'
+    | 'description'
+    | 'level'
+    | 'location'
+    | 'price'
+    | 'sport';
+  placeholder?: string;
+  errors: FieldErrors;
+  watchedValue?: string | number | Date;
+  className?: string;
+};
+
+export const EventInput: React.FC<InputProps> = ({
```

Comment:
Why is this Event input? how is it specific to event? It does look like a regular/generic input

Reply:
Hmm, it is different! But okay, I will need to use IsEventInput for now, and I will also try to consider better implementation.